analytics_data_api/v0/documents.py

```diff
@@ -146,7 +149,13 @@ def get_users_in_course(
             }
             for sort_policy in sort_policies
         ])
-        return search_request.execute()
+        res = search_request.execute()
+        # debugging, to be removed
+        logger.warning(res.__class__.__name__)
+        items = res.hits
+        logger.warning(items[0])
```

Comment:
if it's a different object than you expect are you sure res hits exists and is an array and has something in it?

does the caller do hits and hits[0] and isn't crashing so this must be ok?

Reply:
I think the questionable objects are within `hits`.  I'd expect this class RosterEntry but I'm seeing a generic Hit object instead (I think).